themes/wdfn_theme/layouts/_default/baseof.html

```diff
@@ -1,73 +1,79 @@
 <!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml" lang="en-us" xml:lang="en-us">
-  <head>
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+<head>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
     <meta charset="utf-8">
     {{ hugo.Generator }}
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">
     <link rel="icon" href="{{ $.Site.BaseURL }}img/main/favicon.ico" type="image/x-icon">
-    <link rel="stylesheet" href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/styles/{{ with .Site.Params.SyntaxHighlightTheme }}{{ . }}{{ else  }}solarized_dark.min.css{{ end }}">
+    <link rel="stylesheet"
+          href="//cdnjs.cloudflare.com/ajax/libs/highlight.js/8.4/styles/{{ with .Site.Params.SyntaxHighlightTheme }}{{ . }}{{ else  }}solarized_dark.min.css{{ end }}">
+    <script src="{{ $.Site.BaseURL }}scripts/uswds-init.min.js"></script>
```

Comment:
This is the only new thing in this file. Everything thing else is reformatting. This bit of script was added by USWDS to prevent the banner from flashing (appearing open before loading is done).